python/llm/src/bigdl/llm/optimize.py

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from .transformers import ggml_convert_quant
+from bigdl.llm.ggml.quantize import ggml_tensor_qtype
+from bigdl.llm.utils.common import invalidInputError
+
+def optimize_model(model, load_in_low_bit, optimize_atten=True):
```

Comment:
change to optimize_model(model, low_bit='sym_int4', optimize_llm=True)

Reply:
ok. I'll also change `optimize_llm=False`, since it may not apply to all models (e.g. latest transformer version)

Reply:
> I'll also change `optimize_llm=False`, since it may not apply to all models (e.g. latest transformer version)

You should keep its behavior the same as `from_pretrained`, which I believe defaults to True.

Reply:
> `from_pretrained` defaults to True and do optimizations aggressively.

I think we need to do the same as it helps llm models